Tests/IntegrationTests/Rest/DotNetIntegrationTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Threading.Tasks;
+using FluentAssertions;
+using Xunit;
+
+namespace DK.Gov.Oio.Idws.IntegrationTests.Rest
+{
+    public class DotNetIntegrationTests : IntegrationTestBase
+    {
+        private string _requestUri = "DotNetIntegrationTests.";
+        
+        public DotNetIntegrationTests()
+        {
+            var dotNetWspConfiguration = Configuration.BuildDotNetWspConfiguration();
+            ConfigureWscAndSts(dotNetWspConfiguration);
+        }
+        
+        [Fact]
+        public async Task TestSystemUserScenario()
+        {
+            _requestUri += "TestSystemUserScenario";
+            ConfigureSystemUserScenario();
+            await TestHttpClient();
+        }
+
+        [Fact]
+        public async Task TestLocalTokenScenario()
+        {
+            _requestUri += "TestLocalTokenScenario";
+            ConfigureLocalTokenScenario();
+            await TestHttpClient();
+        }
+
+        private async Task TestHttpClient()
+        {
+            var client = await CreateHttpClientWithIssuedToken();
+            var response = await client.GetAsync(_requestUri);
+            var responseString = await response.EnsureSuccessStatusCode().Content.ReadAsStringAsync();
+            
+            responseString.Should().StartWith(
+                $"Requested at {Configuration.RestWspConfiguration.Endpoint}{_requestUri}\nAuthenticationType: OioIdws\r\nClaims:");
```

Comment:
Giver det mening også at asserte på de modtagede claims?

Reply:
Jeg tror ikke det er så relevant for det vi tester her. Måske @mdbech har en anden holdning?

Reply:
Nej, det giver ikke rigtig mening, da det vil afhænge af OIO SAML versionen for WSP'en, så det vil kunne skifte.